src/main/java/jakarta/inject/package-info.java

```diff
@@ -132,7 +132,7 @@
  * product of the number of unit tests and the number of dependencies.
  *
  * <p><b>This package provides dependency injection annotations that enable
- * portable classes</b>, but it leaves external dependency configuration up to
+ * portable classes</b>, But it leaves external dependency configuration up to
```

Comment:
This is wrong, please remove.

Reply:
Yes It's a typo on my side i will remove it for sure

Reply:
Fix on the last commit